packages/atomic-angular/package.json

```diff
@@ -37,8 +37,9 @@
     "karma-jasmine": "5.1.0",
     "karma-jasmine-html-reporter": "1.7.0",
     "ncp": "2.0.0",
-    "ng-packagr": "16.0.1",
-    "typescript": "4.9.5",
-    "@coveo/headless": "2.30.1"
+    "ng-packagr": "16.2.3",
+    "typescript": "5.1.6",
+    "@coveo/headless": "2.30.1",
+    "jest": "29.6.4"
```

Comment:
Do we really require Jest in this particular package 🤔 ?

I don't think there's any test in this particular directory (it's instead tested through samples/angular)